ui/src/app/components/apps-panel/utils.tsx

```diff
@@ -25,4 +38,55 @@ export const appAssets: AppAssets[] = [
     logo: rStudioLogo,
     icon: rStudioIcon,
   },
+  {
+    appType: UIAppType.CROMWELL,
+    logo: cromwellLogo,
+    icon: cromwellIcon,
+  },
 ];
+
+// TODO replace with better defaults
+export const defaultCromwellConfig: CreateAppRequest = {
+  appType: AppType.CROMWELL,
+  kubernetesRuntimeConfig: {
+    numNodes: 1,
+    machineType: DEFAULT_MACHINE_NAME,
+    autoscalingEnabled: false,
+  },
+  persistentDiskRequest: {
+    size: 10,
```

Comment:
Is 10 too small for this app? I usually use 500GB, but I never test 10 before.

Reply:
I have no idea what values are correct here :)

If you have better defaults, I'm happy to use them, in this PR or later.

Reply:
Update: Marcus is checking on what disk size is appropriate